src/gov/nasa/worldwind/formats/georss/GeoRSSParser.java

```diff
@@ -319,7 +319,7 @@ private static Renderable makePolygonShape(Node node, Iterable<Node> attrs)
             return null;
         }
 
-        ArrayList<LatLon> positions = new ArrayList<LatLon>();
+        ArrayList<LatLon> positions = new ArrayList<>();
```

Comment:
I've been getting the following message regarding this line:
`Error:(322, 53) java: diamond operator is not supported in -source 6
  (use -source 7 or higher to enable diamond operator)`
Did a fresh install of oraclejdk11 and a fresh clone, to no avail.